releasing/plugin-commands-publishing/test/publish.ts

```diff
@@ -613,6 +614,127 @@ test.skip('convert specs with relative workspace protocols to regular version ra
   })
 })
 
+describe('catalog protocol converted when publishing', () => {
```

Comment:
Are we okay with this test being in `releasing/plugin-commands-publishing`? I would guess `pkg-manager/core` isn't the right place for a publishing related test.